### PR TITLE
Completed css-two-column-layout Challenge - Bingle Padilla

### PR DESCRIPTION
**Pull request by codewithbingle**
css-two-column-layout

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+header > p {
+  font-family: karla, sans-serif;
+  font-weight: 400;
+  text-align: center;
+  font-size: small;
+}
+
+.container {
```
Comment (Zhiwutian) [inline css-two-column-layout/styles.css:22]: The container should have a set width on it, for this exercise I would recommend 1200px. Its good to remember to set hard coded values like pixels on the container, and rows and columns should be percentages

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  flex-wrap: wrap;
+}
+
+.row {
+  flex-basis: calc(100% / 2);
+  border: 10px;
+  padding: 30px;
+}
+
+.column {
```
[the remark this answers is not in the document]
Comment (codewithbingle) [reply css-two-column-layout/styles.css:34]: Ok!

File: css-two-column-layout/styles.css (hunk 1 above)
Comment (codewithbingle) [reply css-two-column-layout/styles.css:22]: Noted.

**Comment by codewithbingle** [comment]
css-two-column-layout-update2

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Bingle, it is getting there! I made some changes to the CSS to get you on a better path using display flex on the rows so the columns work correctly. Please let me know if you have questions!

**Comment by codewithbingle** [comment]
css-two-column-layout-update3

**Comment by codewithbingle** [comment]
css-two-column-layout-update4

**Comment by codewithbingle** [comment]
css-two-column-layout-update5

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**